client/src/components/ClientProfile/Information/Information.js

```diff
@@ -240,18 +334,20 @@ export const Info = (props) => {
               >
                 <InputLabel
```

Comment:
As I said before, you should use `common/InputField`, it will shorten your code a little bit.

Reply:
It's optional like snackbar from material UI, you know.

Reply:
But it will take more time for refactoring that I would use for other components.